### PR TITLE
Implement testing of Github Pull Requests

### PR DESCRIPTION
**Pull request by dirkmueller**

### CODE REVIEW

File: scripts/github-status/github-status.rb [hunk 1]
```diff
@@ -84,7 +84,7 @@ def print_pr_sha_info(pull)
         print_pr_sha_info(p)
       end
     else
-      puts "#{pull.number}:#{pull.head.sha}" if pull
+      puts "#{pull.number}:#{pull.head.sha}:#{pull.base.ref}" if pull
```
Comment (MaximilianMeister) [inline scripts/github-status/github-status.rb]: this is the string that ends up in the description right?
is there a way to trim the `sha` a bit here? maybe to 7, as done e.g. by `git rev-parse --short HEAD`

Comment (dirkmueller) [inline scripts/github-status/github-status.rb]: no, we need the full sha here. you're only talking about the jenkins display?

Comment (MaximilianMeister) [inline scripts/github-status/github-status.rb]: yes i mean the jenkins display

Comment (dirkmueller) [inline scripts/github-status/github-status.rb]: There should be a way to trim the jenkins display, but thats unrelated to the change in this place (its in the jenkins job definition)

**Comment by bmwiedemann** [comment]
:+1:

**Pull request merged**